src/composables/useModalSize/index.ts

```diff
@@ -0,0 +1,20 @@
+import {PropType} from 'vue';
```

Comment:
I would not be bothered if you would add `xl` to `useSizes` instead of creating this composable. Indeed `md` does not exist on most of components, but they are still able to use it.